dev-requirements.txt

```diff
@@ -1,2 +1,6 @@
 factory_boy>=3.2,<3.3
+ops>=2.1,<2.2
```

Comment:
Why pinning ops?

Reply:
Mainly because everything else was pinned, I do agree canonical provided libraries probably shouldn't be pinned, should be removed in next commit